packages/ckeditor5-utils/src/dom/scroll.ts

```diff
@@ -292,26 +292,30 @@ function scrollWindowToShowRect<T extends boolean, U extends IfTrue<T>>(
  * @param options.forceScroll When set `true`, the `rect` will be aligned to the top of scrollable ancestors
  * whether it is already visible or not. This option will only work when `alignToTop` is `true`
  */
-function scrollAncestorsToShowRect<T extends boolean, U extends IfTrue<T>>(
+export function scrollAncestorsToShowRect<T extends boolean, U extends IfTrue<T>>(
 	{
 		parent,
 		getRect,
 		alignToTop,
 		forceScroll,
-		ancestorOffset = 0
+		ancestorOffset = 0,
+		limiterElement
```

Comment:
Shouldn't we add this option to `scrollAncestorsToShowTarget()` and use it in the document outline instead of making this helper public?

I mean, this `limiterElement` argument would stay there (it needs docs, BTW) but we wouldn't need to make this low-level API exportable.

Reply:
TBH this choice was purely caused by the functions signatures.

`scrollAncestorsToShowTarget()` has 2 params of which one is optional. I didn't want to enforce passing the nullish `ancestorOffset` to use `limiterElement`. However DO was the only place that used this function and it was passing the `ancestorOffset` anyways, so I guess we can go with your proposal.